agent/tool-scripts/datalog/pprof-datalog

```diff
@@ -56,44 +57,23 @@ else:
         )
         sys.exit(1)
 
-try:
-    from ansible.parsing.dataloader import DataLoader
-    from ansible.inventory.manager import InventoryManager
-except ImportError as exc:
-    logging.error(
-        "[%s]: required Ansible python3 modules not available, '%s'", PROG, exc
-    )
-    sys.exit(1)
+import subprocess  # noqa E402
```

Comment:
Why not just pop all the imports up to the top where they belong? They don't seem to be truly "conditional" since there are only abnormal error exits before them.

Reply:
I have been desperately trying to avoid making changes to these tool datalogs where possible when the changes are cosmetic.  These tools are not heavily used, and very hard to verify they work correctly, and I'd like to eventually drop them entirely.

Historically, this code was contributed by somebody who wanted this data collected by the pbench-agent in a Kubernetes environment, but then they have stopped using the pbench-agent, and this is now "dying on the vine."

Unfortunately, the test for this code keeps failing randomly, impacting development cycles.  So the changes here are trying to be the bare minimum to avoid other bad behaviors.